packages/devtools_app/lib/src/service/resolved_uri_manager.dart

```diff
@@ -1,41 +1,47 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
 import '../shared/globals.dart';
-import 'vm_service_wrapper.dart';
 
 /// Manager for handling package Uri lookup and caching.
 class ResolvedUriManager {
-  Map<String, String?>? _resolvedUrlMap;
-  VmServiceWrapper? service;
+  /// Maps isolate ids to the mappings of resolvedUrls to packageUris
+  ///
+  /// The map keys and values are as follows:
+  /// `_isolateResolvedUrlMap[isolateId][resolvedUrl] = packageUrl`
+  Map<String, Map<String, String?>>? _isolateResolvedUrlMap;
 
   /// Initializes the [ResolvedUriManager]
-  void vmServiceOpened() {
-    _resolvedUrlMap = <String, String?>{};
-  }
+  void vmServiceOpened() =>
+      _isolateResolvedUrlMap = <String, Map<String, String?>>{};
 
   /// Cleans up the resources of the [ResolvedUriManager]
-  void vmServiceClosed() {
-    _resolvedUrlMap = null;
-  }
+  void vmServiceClosed() => _isolateResolvedUrlMap = null;
```

Comment:
_usually_ we'll use the `=>` style for one liners when we actually need to return something, but it doesn't really matter. More of a style preference than a hard guideline.